client/state/hosting/reducer.js

```diff
@@ -5,12 +5,12 @@
 import { keyedReducer, combineReducers } from 'state/utils';
 import { HOSTING_SFTP_USER_UPDATE, HOSTING_SFTP_USERS_SET } from 'state/action-types';
 
-const sftpUsers = ( state = {}, { type, users } ) => {
+export const sftpUsers = ( state = {}, { type, users } ) => {
 	if ( type === HOSTING_SFTP_USERS_SET ) {
 		return users;
 	}
 
-	if ( type === HOSTING_SFTP_USER_UPDATE ) {
+	if ( type === HOSTING_SFTP_USER_UPDATE && Array.isArray( state ) ) {
```

Comment:
Nice Array check! I think this was firing a JS error issues during AT in https://github.com/Automattic/wp-calypso/pull/37983#discussion_r351510855

Reply:
Yup, this was the cause! But I added a workaround in https://github.com/Automattic/wp-calypso/pull/38045 that prevents this error by no requesting the SFTP users if site is not Atomic:

https://github.com/Automattic/wp-calypso/pull/38045/files#diff-5bf5400e6559993b2a6f078899135096R83